torch/csrc/api/src/nn/modules/normalization.cpp

```diff
@@ -0,0 +1,45 @@
+#include <torch/nn/modules/normalization.h>
+
+#include <torch/cuda.h>
+#include <torch/utils.h>
+#include <torch/nn/init.h>
+
+#include <ostream>
+#include <utility>
+
+namespace F = torch::nn::functional;
+
+namespace torch {
+namespace nn {
+
+LayerNormImpl::LayerNormImpl(const LayerNormOptions& options_) : options(options_) {
+  reset();
+  if (options.elementwise_affine()) {
+    torch::nn::init::ones_(weight);
+    torch::nn::init::zeros_(bias);
+  }
```

Comment:
This code block
```cpp
  if (options.elementwise_affine()) {
    torch::nn::init::ones_(weight);
    torch::nn::init::zeros_(bias);
  }
```
should be at the end the `reset()` function, because the expectation is that all initialization logic lives in the `reset()` function :D

Reply:
alright